lib/popover.js

```diff
@@ -169,6 +169,14 @@ export const Popover = ({ align, renderLabel, views = {}, initialView, startOpen
     </PopoverContainer>
   );
 };
+
+Popover.defaultProps = {
+  /* there's a good chance this isn't the right default, but it maintains backwards compatability with the community site,
+  from a time when Popover was not actually respecting startOpen and was passing true through to the underlying PopoverContainer
+  all the time */ 
+  startOpen: true,
```

Comment:
I'm a bit worried about this... since most popovers shouldn't startOpen. If this starts working as intended, will we suddenly have an open popover nightmare site?

Granted that's also something we can probably test locally before updating the package version in prod, right?